src/transactions/test/InvokeHostFunctionTests.cpp

```diff
@@ -1338,14 +1338,15 @@ TEST_CASE("contract storage", "[tx][soroban]")
         auto autobump = enableAutobump();
         REQUIRE(autobump > 1);
 
-        // Check that manual bump doesn't go over max
+        // Check that attempting to bump past max ledger results in error
         put("key", 0, ContractDataDurability::PERSISTENT);
-        bump("key", ContractDataDurability::PERSISTENT, UINT32_MAX);
+        bumpWithFootprint("key", UINT32_MAX, contractKeys,
+                          {contractDataKey(contractID, makeSymbolSCVal("key"),
+                                           ContractDataDurability::PERSISTENT, DATA_ENTRY)},
+                          false, ContractDataDurability::PERSISTENT);
```

Comment:
I think this is the right update to this test @sisuresh since the bump past max ledger is no longer allowed by the env?